stanford/roles/cluster/defaults/main.yml

```diff
@@ -26,11 +26,21 @@ CLUSTER_SUBNETS:
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.5.0/24"
     - zone: 'us-west-1c'
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.6.0/24"
+  cache:
```

Comment:
Presumably these two stanzas are to document the existing, but unscripted clusters?

Reply:
correct.